timemachine/fe/single_topology.py

```diff
@@ -178,23 +196,31 @@ def setup_dummy_interactions(
         if all([a in dga for a in idxs]):
             dummy_improper_idxs.append(tuple([int(x) for x in idxs]))
             dummy_improper_params.append(params)
+
+    # certain configuration of chiral states are symmetrizable
+    #
+    #        all allowed geometries              |  disallowed geometry
```

Comment:
To clarify, `all allowed geometries` here means that the chiral restraint can be left on for the entire simulation?

Reply:
they can be left on at all values of lambda (with the exception of one of the bonds ci, cj, ck being broken). Then the restraint needs to be interpolated.